src/hammer-vlsi/defaults.yml

```diff
@@ -817,6 +853,29 @@ sim.submit:
   settings: "vlsi.submit.settings"
   settings_meta: lazycrossref
 
+power.inputs:
```

Comment:
Could we add documentation for all these settings?

Reply:
Done